plugins/com.google.cloud.tools.eclipse.usagetracker/META-INF/MANIFEST.MF

```diff
@@ -9,7 +9,8 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-ActivationPolicy: lazy
 Bundle-Localization: plugin
 Export-Package: com.google.cloud.tools.eclipse.usagetracker
-Require-Bundle: org.eclipse.core.runtime;bundle-version="3.11.1",
+Require-Bundle:  com.google.gson;bundle-version="[2.8.2,3.0)",
```

Comment:
I remember whitespace in `MANIFEST.MF` is important and causes trouble at runtime. Let's fix the leading double-space.

Reply:
done